src/ol3cesium.js

```diff
@@ -11,12 +11,26 @@ goog.require('olcs.VectorSynchronizer');
 
 
 /**
- * @param {!ol.Map} map
+ * @param {!(ol.Map|olcsx.OLCesiumOptions)} arg1 `ol.Map` or options
  * @param {Element|string=} opt_target Target element for the Cesium container.
+ *   Overrides a target set in options.
  * @constructor
  * @api
  */
-olcs.OLCesium = function(map, opt_target) {
+olcs.OLCesium = function(arg1, opt_target) {
```

Comment:
The ol3cesium constructor will definitively allow more and more customisations making it necessary to use the new behaviour.
It was an error to design this constructor this way and we have now the responsibility to fix it before more and more users start relying on it.

It is straightforward to switch to the new way, thus we can have a short deprecation period of 1 'minor' release: release 1.1.0 will have the new behaviour but still accept the old behaviour with deprecation notice stating that the old behaviour will be removed in the 1.2.0 release. At that time, the CHANGES.md will be updated with a note about this change.